microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -747,20 +747,38 @@ To enable diagnostic data collection, go to `edge://settings/privacy`. Turn on t
 
-If you select **Microsoft Editor**, Microsoft Edge sends your typed text and a service token to a Microsoft cloud service over a secure HTTPS connection. The service token doesn't contain any user-identifiable information. A Microsoft cloud service then processes the text to detect spelling and grammar errors in your text. All your typed text that's sent to Microsoft is deleted immediately after processing occurs. No data is stored for any period of time. 
+By default, Edge provides spelling and grammar checking using **Microsoft Editor**. When using **Microsoft Editor**, Microsoft Edge sends your typed text and a service token to a Microsoft cloud service over a secure HTTPS connection. The service token doesn't contain any user-identifiable information. A Microsoft cloud service then processes the text to detect spelling and grammar errors in your text. All your typed text that's sent to Microsoft is deleted immediately after processing occurs. No data is stored for any period of time.   
 
-If you select **Basic** under the **Use writing assistance** toggle in `edge://settings/languages`, the service will perform only local spellchecking on the device, and no data will be sent to the cloud for spellchecking services. 
+To use **Basic** spelling and grammar checking: 
+1. Go to `edge://settings/languages`    
+1. Select the **Basic** setting in **Use writing assitance**
 
-To turn off all spelling and grammar capabilities, turn off the **Use writing assistance** toggle.
+If you select **Basic**, the service will perform only local spellchecking on the device, and no data will be sent to the cloud for spellchecking services. 
+
+To turn off all spelling and grammar capabilities:
+1. Go to `edge://settings/languages`
+1. Turn off the **Use writing assistance** toggle 
+
+#### Languages
+When you add a new language to Microsoft Edge, the browser downloads the dictionary for the new language to the device over HTTPS. The dictionary is used for the basic spellcheck service or for languages that aren't supported by Microsoft Editor spelling and grammar checking. Deleting the language from your Microsoft Edge settings deletes the dictionary from the device. 
  
-When you add a new language to Microsoft Edge, the browser downloads the dictionary for the new language to the device over HTTPS. The dictionary is used for the basic spellcheck service or for languages that aren't supported by Microsoft Editor spelling and grammar checking. Deleting the language from your Microsoft Edge settings deletes the dictionary from the device.
+#### Text prediction 
+
+Microsoft Edge will automatically provide word and sentence predictions in certain text boxes on the web. These predictions are only visible to you and are not inserted into the text box until you press Tab or the Right arrow to accept them.  
+
+To disable text predictions in Microsoft Edge: 
```

Comment:
```suggestion
To turn off text predictions in Microsoft Edge: 
```
Privacy article is for general audience as well as technical audience, and this doc'n is about end-user GUI.  Thus lean toward "turn off" instead of "disable".
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/d/disable-disabled
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/turn-on-turn-off